db/external_sst_file_test.cc

```diff
@@ -3853,91 +3854,24 @@ TEST_P(IngestDBGeneratedFileTest, FailureCase) {
       ASSERT_OK(Put(1, Key(k), "cf1_" + Key(k)));
     }
     ASSERT_OK(Flush(/*cf=*/1));
-    {
```

Comment:
Why did we delete testing on rejection of non-zero seqno when allow_db_generated_files = false?